utils/nonce.go

```diff
@@ -27,7 +27,7 @@ func (u *EpochNonceGenerator) GetNonce() string {
 
 func NewEpochNonceGenerator() *EpochNonceGenerator {
 	return &EpochNonceGenerator{
-		nonce: uint64(time.Now().Unix()) * 1000,
+		nonce: uint64(time.Now().Unix()) * 1000000,
```

Comment:
We need to be careful with this part - if the nonce generation is different to the other libs then the user’s API keys will not work with the other libs. See nodes nonce gen function https://github.com/bitfinexcom/bfx-api-node-util/blob/13534ac35863e90f005281ee12fb8ad41fe2a634/lib/nonce.js#L10